tests/manage/mcg/test_cached_buckets.py

```diff
@@ -0,0 +1,264 @@
+import time
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.bucket_utils import (
+    copy_objects,
+    copy_random_individual_objects,
+    verify_s3_object_integrity,
+    write_random_objects_in_pod,
+    sync_object_directory,
+)
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id
+from ocs_ci.framework.testlib import MCGTest
+
+logger = logging.getLogger(__name__)
+
+
+class TestCachedBuckets(MCGTest):
+    """
+    Tests Noobaa cache bucket caching mechanism
+
+    """
+
+    @polarion_id("OCS-4651")
+    def test_cached_buckets_with_s3_cp(
+        self, bucket_factory, awscli_pod_session, test_directory_setup, mcg_obj, cld_mgr
+    ):
+        """
+        This test make sure caching mechanism works between hub bucket & cache bucket
+        when we have TTL > 0 and we use `s3 cp` to download objects
+
+        """
+        TTL = 300000
+        cache_bucketclass = {
```

Comment:
Create a fixture/setup for this and call where needed, Coz it looks like you are using same bs/bc for all tests